src/builder/Accounts.sol

```diff
@@ -82,6 +102,23 @@ library Accounts {
         }
     }
 
+    function findMorphoPositions(
+        uint256 chainId,
+        address loanToken,
+        address collateralToken,
+        ChainAccounts[] memory chainAccountsList
+    ) internal pure returns (MorphoPositions memory found) {
+        ChainAccounts memory chainAccounts = findChainAccounts(chainId, chainAccountsList);
+        for (uint256 i = 0; i < chainAccounts.morphoPositions.length; ++i) {
+            if (
+                chainAccounts.morphoPositions[i].loanToken == loanToken
+                    && chainAccounts.morphoPositions[i].collateralToken == collateralToken
+            ) {
```

Comment:
Yeah.. I think I have also replied similar comments in above as well. 
I think this have been discussed previously in #morpho channel, I was raising exact concerns about the markets.  
But from my understanding of the signals from product teams, I think they wanted just do all the hardcoding for now and keep the client as simple as down to just `colalteralToken` and `borrowToken` so that's why we reached to this approach.  (trimmed off all the lltv, irm, oracles input)

But also it's not hard to add that support in the future by incorporating a new input parameter from what we see fits in the future.

Reply:
yeah, I think it's reasonable for the client interface to always be `(collateralToken, loanToken)`, but it's still possible for the builder internally to track multiple lending markets.

Like you said when we talked offline, there's a "dead market" phenomenon where a market is deprecated, or we prefer a different one in the product, and yet a user continues to have positions there. We don't want those to disappear... so at some point we'll have to address this possibility, and ideally without adding complexity to the client.